Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -334,6 +334,12 @@ protected theorem add_sub_cancel_left (ha : a ≠ ∞) : a + b - a = b :=
 protected theorem add_sub_cancel_right (hb : b ≠ ∞) : a + b - b = a :=
   (cancel_of_ne hb).add_tsub_cancel_right
 
+protected theorem sub_add_eq_add_sub (hab : b ≤ a) (b_ne_top : b ≠ ∞) (c_ne_top : c ≠ ∞) :
+    a - b + c = a + c - b := by
```

Comment:
This seems to be true for `c = ∞`: both sides are equal to `∞`.

Reply:
Indeed, thanks! I will try to drop the hypothesis.